message_center_compassion/mappings/compassion_mapping.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Flückiger Nathan <nathan.fluckiger@hotmail.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import models, fields, api
+
+
+class CompassionMapping(models.Model):
+
+    _name = "compassion_mapping"
```

Comment:
Convention is to used dotted names. `compassion.mapping`. Please update everywhere accordingly.